api/interservice/authn/tokens.proto

```diff
@@ -8,12 +8,12 @@ option go_package = "github.com/chef/automate/api/interservice/authn";
 
 message CreateTokenReq {
   // Match either a completely empty string or a valid id.
-  string id = 1 [(validate.rules).string.pattern = "^[a-z0-9-]{1,64}$|^$"];
+  string id = 1 [(validate.rules).string.pattern = "^[a-z0-9-_]{1,64}$|^$"];
```

Comment:
Isn't it safer to always put a loose hyphen as the last character in the char class? That way there is less ambiguity. 🤷‍♂

Reply:
If it was ambiguous it wouldn't work 😉

> But go with whatever you think best...

All the same to me, so I'll keep it as-is if you don't mind 😃

Reply:
I meant ambiguous to the reader. 😁 That is, it is just slightly more confusing to parse visually.